java/ql/src/experimental/CWE-532/SensitiveInfoLog.ql

```diff
@@ -0,0 +1,82 @@
+/**
+ * @id java/sensitiveinfo-in-logfile
+ * @name Insertion of sensitive information into log files
+ * @description Writing sensitive information to log files can give valuable guidance to an attacker or expose sensitive user information.
+ * @kind problem
+ * @tags security
+ *       external/cwe-532
+ */
+
+import java
+import semmle.code.java.dataflow.TaintTracking
```

Comment:
`TaintTracking` isn't actually used here - just plain data flow, so this import is a bit misleading. I'd suggest simply importing `semmle.code.java.dataflow.DataFlow` instead.

Reply:
Thanks @aschackmull for the advice. TaintTracking::Configuration is much more simple and effective. I've replaced the custom source class with DataFlow::Configuration having TaintTracking::localTaintStep(node1, node2) as its isAdditionalFlowStep. The code functions the same but is much more elegant.

@luchua-bc